_layouts/page.html

```diff
@@ -9,7 +9,7 @@
                     <div class="entry-header-wrap">  
                         {% if page.banner_image %}
                         <div class="post-thumbnail">
```

Comment:
I dont like the fact that the parent and the child have the same class "post-thumbnail", we should use proper css selectors.